### ISSUE TITLE
Unicode error on piping some cheatsheets to `less`

### ISSUE THREAD

**Issue by OverkillGuy** [chrisallenlane/cheat#289]
### Reproducing

```
$ cheat hub | less
```

produces

```
Traceback (most recent call last):
  File "/usr/local/bin/cheat", line 61, in <module>
    print(colorize(sheet.read(options['<cheatsheet>'])))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2192' in position 308: ordinal not in range(128)
```

when `cheat hub`  works fine on its own.
## Investigation

```
$ file -bi /usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/hub
text/plain; charset=utf-8
$ file -bi /usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/git
text/plain; charset=us-ascii
```

`cheat git | less` works fine, and has a different charset now we're onto something.

Which other files have that charset ?

```
$ cd /;cheat -l | cut -d'/' -f 2- | xargs file -bi
$ cd /;cheat -l | cut -d'/' -f 2- >~/cheat_fileloc.txt
$ paste -d "    " ~/cheat_fileloc.txt ~/cheat_charsets.txt >~/cheats_file-charset.txt
```

Attached is the resulting  [cheats-filecharset.txt](https://github.com/chrisallenlane/cheat/files/393928
/cheats-filecharset.txt)

Which shows 

```
$ grep "utf" ~/cheats-filecharset.txt
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/find text/plain; charset=utf-8
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/hub text/plain; charset=utf-8
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/nkf text/plain; charset=utf-8
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/pdftk text/plain; charset=utf-8
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/readline text/plain; charset=utf-8
usr/local/lib/python2.7/dist-packages/cheat/cheatsheets/sam2p text/plain; charset=utf-8
```

```
$ cheat find | less
Traceback (most recent call last):
  File "/usr/local/bin/cheat", line 61, in <module>
    print(colorize(sheet.read(options['<cheatsheet>'])))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa0' in position 1102: ordinal not in range(128)
```

Found the culprit !
## Mitigation

Either:
-  Change the python code used for encoding/decoding cheatsheets output
-  Conversion of the incriminating utf-8 cheatsheets to ascii

**Comment by OverkillGuy** [comment]
I can try to take a look at the code and work towards a PR (my first PR <3) later in the week

**Comment by sysadmiral** [comment]
_Most_ modern linux systems use utf-8 as the default which would suggest the correct fix would be to change the default file type for `cheat` to utf-8 however maybe not everyone is using utf-8 as the default (like yourself @OverkillGuy).

So perhaps a better way to solve this is to use the LANG or LOCALE variables to specify the correct value for less to use.

i.e. `cheat hub | LC_ALL=en_GB.UTF-8 less` and replace en_GB with your preferred language.

It's a bit annoying to try and remember which `cheat` sheets need utf-8 so perhaps a wrapper script would be better than a sweeping change?

**Comment by sysadmiral** [comment]
In fact utf-8 is backwards compatible with ascii so permanently setting your locale/lang to utf-8 will fix your problem?

**Comment by chrisallenlane** [comment]
Sorry this took so long to address. This should be resolved as of `cheat` version `2.4.1`. Please update your installation and let me know if it works :smile: 

Closing as resolved.
